.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
-data
\ No newline at end of file
+data
+.nyc_output/
+.idea/
```

Comment:
This can be ignored in the users global gitignore?

Reply:
Github puts them all in the default `.gitignore`, presumably thinking it improves user experience.